### ISSUE TITLE
Missing icons in taskbar

### ISSUE THREAD

**Issue by GeneralPoxter** [mypaint/mypaint#1111]
I have searched for previous missing icons issues as recent as May of this year, but they were closed as a recent update claimed to fix the issues. I also tried the solutions recommended in those issues, but they didn't seem to work.

#### Description of the problem
When opening mypaint, some of the icons in the taskbar do not load properly.
![image](https://user-images.githubusercontent.com/20214615/93001960-0a039200-f501-11ea-82ad-556f779139f6.png)
As seen in the image, the missing icons are replaced by white canvases with an exclamation point in the center.

#### Basic system details

MyPaint version: MyPaint version 2.0.0+gitexport.af5225a
Operating system: Arch
Desktop environment: GNOME

#### Steps to reproduce

1. Start MyPaint
2. Look at the title bar

#### Backtraces or error messages

```
(mypaint:21489): Gtk-WARNING **: 14:01:29.412: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.**
```

**Comment by jplloyd** [comment]
Yes, this is the same problem as in those six issues (#1064, #1068, #1082, #1084, #1085, #1106).

This was indeed fixed back in March, and was part of the 2.0.1 release.

Did you try the workaround for Ubuntu on your Arch install? Keep in mind that if the installation directories are different, that workaround will not work, and would have to be adapted to however the files are installed on your system. At that point it seems easier to me to just build v2.0.1 from source.

Ultimately what I would suggest is to contact the maintainer of the [Arch package for mypaint](https://www.archlinux.org/packages/community/x86_64/mypaint/) and ask them (nicely) to upgrade it to the v2.0.1 release, to fix this issue.

**Comment by GeneralPoxter** [comment]
Thanks, will do.
